### DISCUSSION TITLE
PG bouncer error [May 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2110] [Discord Threads]
> 🙋‍♂️ **antar-das**  
## Nocodb has been throwing errors after upgrading  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/974630373965332490/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### looks like its not related to upgrade.. please check if your credentials are fine  
#  
> 🙋‍♂️ **antar-das**  
#### but it's working find on ^0.84.1 v  
#  
> 💁 **o1lab**  
#### like can you paste project info from there  
#  
> 💁 **o1lab**  
#### from right top menu  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974630374258913291)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
